[2023-01-26T21:46:31.490+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: better_workflow_3.train_model_with_best_hyperparameters_f1_task manual__2023-01-26T21:42:31.315906+00:00 [queued]>
[2023-01-26T21:46:31.535+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: better_workflow_3.train_model_with_best_hyperparameters_f1_task manual__2023-01-26T21:42:31.315906+00:00 [queued]>
[2023-01-26T21:46:31.535+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-01-26T21:46:31.535+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 4
[2023-01-26T21:46:31.535+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-01-26T21:46:31.578+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): train_model_with_best_hyperparameters_f1_task> on 2023-01-26 21:42:31.315906+00:00
[2023-01-26T21:46:31.597+0000] {standard_task_runner.py:55} INFO - Started process 49305 to run task
[2023-01-26T21:46:31.631+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'better_workflow_3', 'train_model_with_best_hyperparameters_f1_task', 'manual__2023-01-26T21:42:31.315906+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmpxqs7bva8']
[2023-01-26T21:46:31.650+0000] {standard_task_runner.py:83} INFO - Job 17: Subtask train_model_with_best_hyperparameters_f1_task
[2023-01-26T21:46:31.691+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.9/site-packages/***/settings.py:249 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-01-26T21:46:31.892+0000] {task_command.py:388} INFO - Running <TaskInstance: better_workflow_3.train_model_with_best_hyperparameters_f1_task manual__2023-01-26T21:42:31.315906+00:00 [running]> on host 69914ba18a5b
[2023-01-26T21:46:32.057+0000] {taskinstance.py:1507} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=baahmedabdessamad@gmail.com
AIRFLOW_CTX_DAG_OWNER=abdessamad
AIRFLOW_CTX_DAG_ID=better_workflow_3
AIRFLOW_CTX_TASK_ID=train_model_with_best_hyperparameters_f1_task
AIRFLOW_CTX_EXECUTION_DATE=2023-01-26T21:42:31.315906+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-26T21:42:31.315906+00:00
[2023-01-26T21:46:32.570+0000] {logging_mixin.py:137} WARNING - 2023/01/26 21:46:32 WARNING mlflow.utils.git_utils: Failed to import Git (the Git executable is probably not on your PATH), so Git SHA is not available. Error: Failed to initialize: Bad git executable.
The git executable must be specified in one of the following ways:
    - be included in your $PATH
    - be set via $GIT_PYTHON_GIT_EXECUTABLE
    - explicitly set via git.refresh()

All git commands will error until this is rectified.

This initial warning can be silenced or aggravated in the future by setting the
$GIT_PYTHON_REFRESH environment variable. Use one of the following values:
    - quiet|q|silence|s|none|n|0: for no warning or exception
    - warn|w|warning|1: for a printed warning
    - error|e|raise|r|2: for a raised exception

Example:
    export GIT_PYTHON_REFRESH=quiet
[2023-01-26T21:47:02.521+0000] {logging_mixin.py:137} WARNING - Registered model 'nba-investment-model-f1' already exists. Creating a new version of this model...
[2023-01-26T21:47:02.592+0000] {logging_mixin.py:137} WARNING - 2023/01/26 21:47:02 INFO mlflow.tracking._model_registry.client: Waiting up to 300 seconds for model version to finish creation.                     Model name: nba-investment-model-f1, version 18
[2023-01-26T21:47:02.593+0000] {logging_mixin.py:137} WARNING - Created version '18' of model 'nba-investment-model-f1'.
[2023-01-26T21:47:02.632+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-01-26T21:47:02.660+0000] {taskinstance.py:1318} INFO - Marking task as SUCCESS. dag_id=better_workflow_3, task_id=train_model_with_best_hyperparameters_f1_task, execution_date=20230126T214231, start_date=20230126T214631, end_date=20230126T214702
[2023-01-26T21:47:02.747+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-01-26T21:47:02.793+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
